testdata/comments.out.go

```diff
@@ -0,0 +1,31 @@
+package test
+
+// docstring should wrap to 80 chars test1 test2 test3 test4 test5 test6 test7
```

Comment:
Why is this kind of output desirable? (specifically keeping the newline after `test2`?) The vast majority of our comments use `//` and have multi-line sentences.

Reply:
My concern is that we're trying to fix a non-problem and enforcing too-rigid rules in the process. The intent of the style guide is for comments to be readable, not to serve as some kind of lawyerly document that requires strict adherence. This wrapping that happens here makes them less readable in most cases.

Reply:
I added these changes because I would get comments on my PRs saying to wrap long lines. I can keep my own copy of crlfmt with these changes and revert them on the main branch if you prefer. Just let me know.